### ISSUE TITLE
 Fetal foot inversion

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7364]
**Preferred term label:**
 Fetal foot inversion

Question -- is this esssentially the same thing as [Talipes equinovarus HP:0001762 ](https://hpo.jax.org/app/browse/term/HP:0001762) or is there a need for a specifically fetal term?

See https://diagnosticpathology.biomedcentral.com/articles/10.1186/s13000-019-0853-x/tables/1

**Comment by PaulNSchofield** [comment]
In principle yes it is the same thing, but I think that it might need a parent of "fetal foot inversion" as fetal foot inversion is also part of "metatarsus varus" (inversion and adduction of the forefoot alone)and . Might also consider "positional talipes equinovarus" as another sibling which is a differential etiology or rather "talipes equivarus" is etiologically agnostic..

**Comment by PaulNSchofield** [comment]
Further digging suggests that  fetal foot inversion is never found as a single phenotype and is always associated with adduction, supination and varus. IE Talipes equinovarus,  and never as an isolated  phenotype, due to the structure of the foot and ankle. A class of "fetal foot malposition" might be a more useful term with Talipes equivarus, metatarsus varus ( adductus)  and positional talipes equivarus as children. It is part of a fetal phenotype though and is often diagnosed in utero by US.

**Comment by PaulNSchofield** [comment]
**Preferred term label**:
Fetal foot inversion
**Synonyms:**
related: fetal foot supination
**Definition**
Malposition of the foot originating during development involving inversion at the region of the metacarpals or more usually the subtalar joint. Formally this is not supination but the term is often used to describe this malformation. PMID: 30450690
**Parent term**
would be Abnormal fetal skeletal morphology HP:0025662 but as this is restricted to phenotypes only visible during the fetal period the probably Positional foot deformity HP:0005656
**Diseases characterised by this term:**
OMIM: 619099 PMID: 31299979

**Comment by monicacecilia** [comment]
From workshop on 2022-04-05: 
- The term ‘foot inversion’ does not seem to be widely used elsewhere. Instead, many use talipes equinovarus (adduction of the forefoot, inversion of the heel and plantar flexion of the forefoot and ankle).
- There may not be a need to add as a separate term, but rather add as a synonym.

**Comment by monicacecilia** [comment]
Hi @PaulNSchofield - question from our workshop participants: 
- Would a synonym be okay in this case? Else, do you have evidence regarding differences between the two entities that justify the new term?

**Comment by PaulNSchofield** [comment]
Hi Monica,

So supination only strictly occurs at the proximal radioulnar joint. The foot cannot supinate in the true sense of the arm and the normal movement in the foot often called supination involves plantar flexion, inversion, and adduction . Inversion and eversion  refer to tilting of the sole of the plantar aspect of the foot towards, or away from, the midline of the body respectively. If there is inversion this occurs at the subtalar joint. The presence of a fixed inversion is often referred to as varus, and the presence of a fixed eversion is referred to as valgus. However the term supination is incorrectly used very widely and you will find it in many papers and textbooks. 

This is discussed here: https://doi.org/10.1054/foot.1999.0502 in great detail. There are some differences in usage depending on the exact position and axis of the two, again this is discussed in the paper. This might I think justify the new class.

However with the widespread use of the term supination where just inversion is meant I wonder if we use some type of “inexact/related  synonym “ but I do think that the preferred label for this pathological condition should be inversion. So the question is whether to simply add it as a sibling of supination, deprecate supination and add it as an inexact/related synonym to a new class for inversion, or to just change the label in the existing class and add the inexact/related synonym of supination ?

I think the paper explains most of the issues.


Cheers, P.

**Comment by pnrobinson** [comment]
@PaulNSchofield for now I made this a narrow synonym of talipes equinovarus and extended the comment as follows:

Clubfoot is a complex, multifactorial deformity with genetic and intrauterine factors. One popular theory postulates that a clubfoot is a result of intrauterine maldevelopment of the talus that leads to adduction and plantarflexion of the foot. On radiographic projection a clubfoot can be noted as parallel axes of talus and calcaneus. Fetal foot inversion refers to malposition of the foot originating during development involving inversion at the region of the metacarpals or more usually the subtalar joint and represents the prenatal sonographic equivalent of talipes equinovarus.

I thiink we will need an onsite hackathon to get any further.
